connectors/data.json

```diff
@@ -0,0 +1,138 @@
+            "get_email_templates": []
+        },
+        "configurations": [
+            {
+                "id": 1,
+                "config_id": "88c3d39c-2fa9-4731-b00d-29815008f17c",
+                "status": 1,
+                "name": "localhost-postfix",
+                "default": true,
+                "config": {
+                    "host": "localhost",
+                    "port": "25",
+                    "useTLS": false,
+                    "timeout": 10,
+                    "password": "",
+                    "default_from": "noreply@example.com"
+                },
+                "remote_status": [],
+                "health_status": [],
+                "connector": 12,
+                "agent": "a4428b2391c9fba4ddf432766be2e3d2",
+                "teams": []
+            }
+        ],
+        "dataImports": [],
+        "rpm_name": "cyops-connector-smtp-2.4.2",
+        "install_mode": "rpm",
+        "installer_path": "cyops-connector-smtp-2.4.2",
+        "ingestionPlaybooks": {
+            "collections": [],
+            "exported_tags": [],
+            "globalVariables": []
+        }
+    },
+    {
+        "name": "symantec-atp",
+        "label": "Symantec ATP",
+        "category": "Threat Protection",
+        "description": "Symantec Advanced Threat Protection (ATP) performs the critical security tasks that detect, protect and respond to threats to your network.",
+        "publisher": null,
+        "version": "2.1.0",
+        "operation_roles": {
+            "get_appliance_information": [],
+            "get_events": [],
+            "get_incidents": [],
+            "get_incidentevents": [],
+            "get_command_status": [],
+            "cancel_command": [],
+            "command_result": [],
+            "isolate_endpoint": [],
+            "rejoin_endpoint": [],
+            "get_incident_comment": [],
+            "add_incident_comment": [],
+            "close_incident": [],
+            "get_file_from_endpoint": [],
+            "delete_endpoint_file": [],
+            "endpoint_recorder_search": [],
+            "endpoint_eoc_search": [],
+            "get_entities": [],
+            "get_domain_entities": [],
+            "get_domain_instance": [],
+            "get_domain_instance_by_domain_name": [],
+            "get_endpoint_entities": [],
+            "get_endpoint_instances": [],
+            "get_specific_endpoint_instances": [],
+            "get_file_entities": [],
+            "get_file_instances": [],
+            "get_file_entity_by_sha2": [],
+            "create_blacklist_policies": [],
+            "get_blacklist_policies": [],
+            "update_blacklist_policy_comment": [],
+            "delete_blacklist_policy": [],
+            "execute_sandbox_commands": [],
+            "get_sandbox_command_status": []
+        },
+        "configurations": [],
+        "dataImports": [],
+        "install_mode": "tgz",
+        "installer_path": "symantec-atp_2.1.0.tgz"
```

Comment:
Why .tgz binary file is included in this SP?

Reply:
Its auto-created .. need to check with Akash

Reply:
okey